### ISSUE TITLE
Support for nested data classes

### ISSUE THREAD

**Issue by WillAvudim** [cs-cordero/py-ts-interfaces#1]
I'm trying to nest one data class in another:

```
@dataclass
@dataclass_json
class A(Interface):
  member: int

@dataclass
@dataclass_json
class B(Interface):
  inner_list: List[A]
```

This fails with 

> File "project/lib/python3.7/site-packages/py_ts_interfaces/parser.py", line 102, in helper
> type_value = TYPE_MAP[node.name]
> KeyError: 'A'

Are nested classes supported or it is not supposed to work?

**Comment by cs-cordero** [comment]
@WillAvudim Thanks for the issue report!  It looks like it is currently unsupported, but it definitely could be.

**Comment by cs-cordero** [comment]
Closed by #2  🎉 Thanks @AadamZ5 

Please let me know if you have any issues nesting Interface data classes!
